Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -378,7 +383,10 @@ class BrowserViewController: UIViewController {
 
     private func updateInContentHomePanel(url: NSURL?) {
         if !urlBar.isEditing {
-            if url == AppConstants.AboutHomeURL {
+            //edit to get only the beginning url!!
+            var aboutHomeEdited: Array = AppConstants.AboutHomeURL.absoluteString!.componentsSeparatedByString("#")
+            var urlEdited: Array = url!.absoluteString!.componentsSeparatedByString("#")
+            if urlEdited[0] == aboutHomeEdited[0] {
```

Comment:
Check out [isReaderModeURL](https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Reader/ReaderModeUtils.swift#L57) as more robust way of checking this. For now, it's probably good enough to create a similar helper method `isAboutHomeURL()` that's inside of `BrowserViewController`.